pkgs/top-level/python-packages.nix

```diff
@@ -29044,6 +29044,8 @@ EOF
     };
   };
 
+  first = callPackage ../development/python-modules/first.nix {};
```

Comment:
My opinion is that, while we diverge on certain matters from the general guidelines in Nixpkgs, we should try and stick to them as much of possible.